oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -36,12 +38,29 @@ fn logger(pretty_name: &str, runtime: RuntimeRef, reader: ChannelReader) -> Resu
         let _ = runtime.wait_on_channels(&[Some(&reader)]);
 
         if let Some(message) = runtime.channel_read(&reader)? {
-            let message = String::from_utf8_lossy(&message.data);
-            info!("{} LOG: {}", pretty_name, message);
+            match LogMessage::decode(&*message.data) {
+                Ok(msg) => info!(
+                    "{} LOG: {} {}:{}: {}",
+                    pretty_name,
+                    to_level_name(msg.level),
+                    msg.file,
+                    msg.line,
+                    msg.message
+                ),
+                Err(error) => error!("{}", error),
```

Comment:
Maybe add some context to this message, e.g. "could not parse LogMessage" or similar. Perhaps copy the structure from the C++ log node?

Reply:
Good point! Done.